opensrp-reveal/src/main/java/org/smartregister/reveal/presenter/ListTaskPresenter.java

```diff
@@ -278,13 +280,19 @@ public void onMapReady() {
     }
 
     public void onMapClicked(MapboxMap mapboxMap, LatLng point) {
+        double currentZoom = mapboxMap.getCameraPosition().zoom;
+        if (currentZoom < MAX_SELECT_ZOOM_LEVEL) {
+            Log.w(TAG, "onMapClicked Current Zoom level" + currentZoom);
```

Comment:
Should this also show some kind of toast message to the user.

Right now they would not be sure why their clicks are not registered.

Reply:
To be resolved in  [https://github.com/OpenSRP/opensrp-client-reveal/issues/38](https://github.com/OpenSRP/opensrp-client-reveal/issues/38)